### PR TITLE
luci-app-advanced-reboot: Add Simplified Chinese translation

### PR DESCRIPTION
**Pull request by imwsy**

### CODE REVIEW

**Comment by hnyman** [comment]
what is that proposed change to Makefile? 
I have never seen "PO = something" in a LuCI app Makefile, I think.

And you have not signed off your commit, like you should have. (See contributing.md)

File: applications/luci-app-advanced-reboot/Makefile [hunk 1]
```diff
@@ -14,6 +14,7 @@ LUCI_DESCRIPTION:=Provides Web UI (found under System/Advanced Reboot) to reboot
 LUCI_DEPENDS:=+luci
 LUCI_PKGARCH:=all
 PKG_RELEASE:=23
+PO = luci-app-advanced-reboot
```
Comment (yousong) [inline applications/luci-app-advanced-reboot/Makefile]: This is something only needed for old build code.  Please remove this change

**Comment by yousong** [comment]
@imwsy , if you are still interested in getting this merged, please resolve conflict and update your local branch.  Thank you

**Comment by dobetter** [comment]
???

**Comment by feckert** [comment]
No reaction from pr creator.

**Pull request closed without merging**